glustercli/cmd/snapshot-info.go

```diff
@@ -47,6 +47,7 @@ func snapshotInfoDisplay(snap api.SnapGetResp) {
 	fmt.Println("Snapshot Volume ID:", vol.ID)
 	fmt.Println("State:", vol.State)
 	fmt.Println("Origin Volume name:", snap.ParentVolName)
+	fmt.Print("Origin Volume Capacity: ", vol.Capacity, "MB\n")
```

Comment:
@harigowtham need to call `HumanReadable` here as well,

Reply:
`Origin Volume Capacity` should be `Snapshot Volume Capacity` right?

Reply:
Done